src/components/MainSidebar/YourDimensionsPanel/useYourDimensions.js

```diff
@@ -2,22 +2,25 @@ import { useDataQuery } from '@dhis2/app-runtime'
 import { useEffect, useState } from 'react'
 import { DIMENSION_LIST_FIELDS } from '../DimensionsList/index.js'
 
+const yourDimensionsResource = 'dimensions'
+// Fixed filter on org units for 2.38 ?
+const yourDimensionsFilter = 'dimensionType:eq:ORGANISATION_UNIT_GROUP_SET'
```

Comment:
Perhaps we should use `SCREAMING_SNAKE_CASE` for these string constants, as we do elsewhere?

Reply:
Of course.
Fixed.